### PR TITLE
improved meshmap mobile swiper

### PR DESCRIPTION
**Pull request by ShravaniAK**
**Description**

This PR fixes #

**Notes for Reviewers**
resolved meshmap mobile swiper bug , and improved the mobile view style .

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit b1403cc2fb1df1efd80e70aef91abf54a026dd7d at: https://64c894f2f1529b3aaa970ef3--layer5.netlify.app

**Comment by Ghat0tkach** [comment]
Hey @ShravaniAK 



Let's discuss it on the websites call.
Please add this as an agenda item in the meeting minutes, if you would :)

https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima

File: src/sections/Meshmap/Meshmap-design/Meshmap_Mobile_swiper/mobile-swiper.style.js [hunk 1]
```diff
@@ -1,143 +1,177 @@
...
+      font-size: 0.8rem;
     }
-
   }
-  @media (max-width:470px){
-    .card{
-      /* height:50vh; */
+  @media (max-width: 490px) {
+    .card {
+      height: 55vh;
```
Comment (thisiskaransgit) [inline src/sections/Meshmap/Meshmap-design/Meshmap_Mobile_swiper/mobile-swiper.style.js:106]: Thanks @ShravaniAK, the reduced height of the cards looks good on mobile view. Why were the formatting changes pushed alongside? please revert them.

**Comment by l5io** [comment]
🚀 Preview for commit 3820159f7725ed15732b1e5f6fceb5bbd1f5eb2b at: https://64e0c0b844ad10502154ac02--layer5.netlify.app

**Comment by Chadha93** [comment]
@ShravaniAK :bug: 
image

**Comment by ShravaniAK** [comment]
@Chadha93 I dont know why but i am not getting this particular issue,because for me its working fine.
Screenshot 2023-08-24 at 9 24 11 AM

**Comment by Ghat0tkach** [comment]
@ShravaniAK , 
Let's discuss it on the websites call.
Please add this as an agenda item in the meeting minutes, if you would :)
https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima

**Review by Chadha93 — APPROVED** [review]
Lets go @ShravaniAK

**Pull request merged**